linkedlist/doubly/python/DoublyLinkedList.py

```diff
@@ -0,0 +1,270 @@
+"""
+Author: Abdulazeez Abdulazeez Adeshina
+"""
+
+
+class Node:
+    """
+    This represents the template for the node object that is used for the linkedlist.
+    """
+
+    def __init__(self, element):
+        """
+        :param element: This represents the element contained in the Node.
+        :type element: This element can be of any type ranging from strings, numbers, arrays etc.
+        """
+        self.element = element
+        self.previous = None
+        self.next = None
+
+
+class DoublyLinkedList:
+    def __init__(self):
+        self.head = None
+        self.tail = None
+        self.size = 0
+
+    def __len__(self) -> int:
+        """
+        :return: Length of the linkedlist
+        :rtype: int
+        """
+        return self.size
+
+    def __str__(self):
+        linkedlist = ""
+        current  = self.head
+
+        while current and current.next is not None:
+            linkedlist += str(current.element) + "<->"
```

Comment:
@Youngestdev FYI, made changes to the arrow to show the two-pointers characteristic

Reply:
Oh cool!